src/cogent3/core/new_alignment.py

```diff
@@ -3632,6 +3566,8 @@ def majority_consensus(self) -> new_sequence.Sequence:
             pos = CategoryCounter(pos)
             states.append(pos.mode)
 
+        # refactor: design
+        # should this return an Aligned object?
```

Comment:
i think it's ok for it to be a Sequence since it's primarily for summarising alignment column contents and it can do so as a Sequence just fine.